qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,34 @@
   </screenshots>
 
   <releases>
+   <release version="2022.11.25-2" date="2022-11-25">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of November 19</li>
+         </ul>
+       <p>Routing and Navigation:</p>
+         <ul>
+           <li>Corrected bicycle's speed for better routing</li>
+         </ul>
+       <p>Desktop:</p>
```

Comment:
It doesn't make sense here and misleads users. Let's remove desktop word.

Reply:
OK, I have removed the `Desktop` paragraph, and I have moved the content of the paragraph under `Highlights`.